File: Sc.Models/Dtos/Suppliers/Junctions/SupplierOfferingTeamDto.cs
using Sc.Models.Entities.Base;

namespace Sc.Models.Dtos.Suppliers.Junctions
{
    public class SupplierOfferingTeamDto : Entity
    {
        public int SupplierOfferingId { get; set; }
        public SupplierOfferingDto SupplierOffering { get; set; }

        public int SupplierTeamId { get; set; }
        public SupplierTeamDto SupplierTeam { get; set; }
    }
}

File: WebApplication/src/suppliers/dtos/junctions/supplier-offering-team.dto.ts
import { SupplierOfferingDto } from "../supplier-offering.dto";
import { SupplierTeamDto } from "../supplier-team.dto";

export class SupplierOfferingTeamDto {
    id: number;

    supplierOfferingId: number;
    supplierOffering: SupplierOfferingDto;

    supplierTeamId: number;
    supplierTeam: SupplierTeamDto;
}